source/product-overview/mattermost-v11-changelog.md

```diff
@@ -0,0 +1,146 @@
+ - All telemetry support have been removed from Mattermost. The system no longer collects or sends telemetry data to external services. Customers that have the NPS plugin enabled can remove it as it no longer sends the feedback over through telemetry.
+ - Format query parameter requirement in ``/api/v4/config/client`` endpoint has been deprecated.
+ - Deprecated mmctl commands and flags have been removed.
+ - Experimental certificate-based authentication feature has been removed. ``ExperimentalSettings.ClientSideCertEnable`` must be ``false`` to start the server.
+ - Added logic to migrate the password hashing method from bcrypt to PBKDF2. The migration will happen progressively, migrating the password of a user as soon as they enter it; e.g. when logging in or when double-checking their password for any sensitive action. There is an edge case where users might get locked out of their account: if a server upgrades to v11 and user A logs in (i.e., they need to enter their password), and then the server downgrades to v10.12 or previous, user A will no longer be able to log in. In this case, admins will need to manually reset the password of such users, through the system console or through the ``mmctl user reset-password [users]`` command.
+ - ``/api/v4/teams/{team_id}/channels/search_archived`` has been deprecated in favour of ``/api/v4/channels/search`` with the deleted parameter.
+ - Changed default database connection pool settings: changed ``MaxOpenConns`` from 300 to 100 and ``MaxIdleConns`` from 20 to 50, establishing a healthier 2:1 ratio for better database connection management.
+ - Separate notification log file has been deprecated. If admins want to continue using a separate log file for notification logs, they can use the ``AdvancedLoggingJSON`` configuration. An example configuration to use is:
+
+ ```
+   {
+     "LogSettings": {
+       "AdvancedLoggingJSON": {
+         "notifications_file": {
+           "type": "file",
+           "format": "json",
+           "levels": [
+               {"id": 300, "name": "NotificationError"},
+               {"id": 301, "name": "NotificationWarn"},
+               {"id": 302, "name": "NotificationInfo"},
+               {"id": 303, "name": "NotificationDebug"},
+               {"id": 304, "name": "NotificationTrace"}
+           ],
+           "options": {
+               "filename": "notifications.log",
+               "max_size": 100,
+               "max_age": 0,
+               "max_backups": 0,
+               "compress": true
+           }, 
+           "maxqueuesize": 1000 
+         }
+       }
+     }
+   }
+ ```
+
+```{Important}
+If you upgrade from a release earlier than v10.10, please read the other [Important Upgrade Notes](https://docs.mattermost.com/administration-guide/upgrade/important-upgrade-notes.html).
+```
+
+### Improvements
```

Comment:
We should announce general FIPS availability here -- @esarafianou, do we have any separate documentation or announcement we could link out to?

Reply:
https://github.com/mattermost/docs/pull/8420 is in progress